### PR TITLE
add faq for 2 common issues and add tool to quickly figure out required syscalls

### PR DESCRIPTION
**Pull request by dafang**
- Add an FAQ file to document common issues.
- Add utility command test code to quickly identify system calls.

### CODE REVIEW

**Comment by dafang** [comment]
@Yeuoly

**Comment by Yeuoly** [comment]
Great works!

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Here's the translated FAQ document in English, formatted in markdown for better readability:
```
Comment (Yeuoly) [inline FAQ.md:1]: Is this a mistake? I guess it's response from a LLM, and it should be deleted

**Review by Yeuoly — CHANGES_REQUESTED** [review]

File: FAQ.md (hunk 1 above)
Comment (dafang) [reply FAQ.md]: Yes, fixed. @Yeuoly

**Review by Yeuoly — APPROVED** [review]
LGTM

**Comment by Yeuoly** [comment]
Please add my wechat: Yeuoly, I will invite you to our contributor group.

**Pull request merged**